CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -749,6 +750,15 @@ local function CreateDropDown(dropDownStringTable, startPosition, settingsHub)
 		this.Selections = {}
 		this.SelectionInfo = {}
 
+		local vrEnabled = UserInputService.VREnabled
```

Comment:
Hm, it's in UpdateDropDownList(). I think this gets called again at a useful time, but I'll double-check that.